<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddTaskFormTudu</class>
 <widget class="QDialog" name="AddTaskFormTudu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Task</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>521</width>
     <height>361</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>519</width>
      <height>359</height>
     </rect>
    </property>
    <widget class="QLineEdit" name="taskTitle">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>14</y>
       <width>401</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 63 19pt &quot;URW Gothic&quot;;</string>
     </property>
     <property name="placeholderText">
      <string>Add Title</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="taskDesc">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>501</width>
       <height>231</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="tuduTaskSave">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>320</y>
       <width>501</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="taskPriority">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>30</y>
       <width>86</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string extracomment="0">High</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string extracomment="1">Medium</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string extracomment="2">Low</string>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
